test/unit/testVariables.ts

```diff
@@ -63,3 +63,30 @@ export const decrypterJSON = JSON.stringify({
     'base64:A3yU8aavNj4LJ97eFAaYpU97q70oSogzBZAlo5tj/1Kj',
   bobSecretKeyBytes: 'base64:dOs3NGmtXJMdjVMaXf3/m5PlAmqwzSGoF9XpyO4LwZk=',
 });
+
+export const TEST_CONTRACT_ADDR = '0x0000000000000000000000000000000000000001';
+export const TEST_CONTRACT_ADDR_2 =
+  '0x0000000000000000000000000000000000000001';
```

Comment:
do we want that TEST_CONTRACT_ADDR have the same (fake) address than TEST_CONTRACT_ADDR_2?

Just asking because in the original condition test file (conditions.test.ts), we had this:

```
const TEST_CONTRACT_ADDR = '0xC36442b4a4522E871399CD717aBDD847Ab11FE88';
const TEST_CONTRACT_ADDR_2 = '0x5dB11d7356aa4C0E85Aa5b255eC2B5F81De6d4dA';
```

Reply:
Nope, that's a mistake. These should be different, i.e. the second one should be `0x0000000000000000000000000000000000000002`

Reply:
Something to think about here is that if mistakenly having them be the same address did not produce a test failure, is the second one actually needed as a constant. May be / may be not i.e. perhaps the second one can just be a local variable.